routes/instagram.js

```diff
@@ -3,7 +3,7 @@ const router = express.Router();
 
 const instagram = require('../mock-database');
 
-const validateInfo = require('../helpers/validator.js')
+const validator = require('../helpers/validator.js')
```

Comment:
if you do 

`const { validateUser } = require('../helpers/validator.js')`

It will extract automatically the 'validateUser' method returned by the map, so you don't need to do 'yourVariable.validateUser'. It's the destructuring feature in ES6

Reply:
not mandatory to do it like this, it's just a nice option that I personally like